examples/slice/CustomError/Client/Program.cs

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) ZeroC, Inc.
+
+using IceRpc;
+using VisitorCenter;
+using ZeroC.Slice; // for the Result<TSuccess, TFailure> generic type
+
+await using var connection = new ClientConnection(new Uri("icerpc://localhost"));
+
+var greeter = new GreeterProxy(connection);
+
+string[] names = [ "", "jimmy", "billy bob", "alice", Environment.UserName ];
+
+foreach (string name in names)
+{
+    Console.Write($"The greeting for '{name}' is ");
+
+    // Use the Dunet-generated MatchXxx methods to process the result and the GreeterError.
+    string message = await greeter.GreetAsync(name).MatchAsync(
```

Comment:
I think it is clear to not use the MatchAsync extension.

```
Result<string, GreeterError> result = await greeter.GreetAsync(name);
result.Match(...);
```

Reply:
Fixed as suggested.